_sass/layouts/_header.scss

```diff
@@ -2,7 +2,7 @@
 
 header.header-section {
   zoom: 1;
-  margin: 30px 0px;
+  margin: 14px 0px;
```

Comment:
Let's remove this and put back the older header spacing. (I think this is the header area spacing, if that's wrong, then disregard, and let's edit the right margin/padding)